guides/v2.2/migration/migration-plan.md

```diff
@@ -17,63 +17,63 @@ Migration is a perfect moment to make serious changes and get your site ready fo
 
-* Install Magento 2.x (with all modules of this release) and the Data Migration Tool on a system that meets the [Magento system requirements]
+*  Install Magento 2.x (with all modules of this release) and the Data Migration Tool on a system that meets the [Magento system requirements]
 
-* Make your custom adjustments to the Data Migration Tool code in case you do not need to migrate some data (like CMS Pages, Sales Rules, etc.) or want to convert your Magento customization during migration. Read the Data Migration Tool's [Technical Specification] to better understand how migration works from inside
+*  Make your custom adjustments to the Data Migration Tool code in case you do not need to migrate some data (like CMS Pages, Sales Rules, etc.) or want to convert your Magento customization during migration. Read the Data Migration Tool's [Technical Specification] to better understand how migration works from inside
 
 ## Step 3: Dry run
 
 Before you start migration on the production environment, it would be best to go through all the migration steps on your testing environment.
 
 In such migration testing, follow these steps:
 
-* Copy your Magento 1 store to a staging server
+*  Copy your Magento 1 store to a staging server
 
-* Fully migrate the replicated Magento 1 store to Magento 2
+*  Fully migrate the replicated Magento 1 store to Magento 2
 
-* Thoroughly test your new store
+*  Thoroughly test your new store
 
 ## Step 4: Start your migration
 
 1. Make sure that the Data Migration Tool has a network access to connect to Magento 1 and Magento 2 databases. Open the corresponding ports in your firewall.
 
-2. Stop all activities in the Magento 1.x Admin Panel, except for order management, such as shipping, creating invoice, credit memos, etc (the list of allowed activities can be extended by adjusting settings of the Delta mode in the Data Migration Tool). **Note:** such activities must not be resumed until your Magento 2 store goes live.
+1. Stop all activities in the Magento 1.x Admin Panel, except for order management, such as shipping, creating invoice, credit memos, etc (the list of allowed activities can be extended by adjusting settings of the Delta mode in the Data Migration Tool). **Note:** such activities must not be resumed until your Magento 2 store goes live.
 
-3. We recommend to stop all Magento 1.x cron jobs.
+1. We recommend to stop all Magento 1.x cron jobs.
 
    Still, if some jobs are required to run during migration, make sure they do not create new database entities or change the existing ones in the way that such entities cannot be processed by the Delta mode.
 
    For example: the `enterprise_salesarchive_archive_orders` cron job moves old orders to archive. Running this job during migration is safe because the Delta mode takes the job into account and thus properly processes the archived orders.
 
 {:start="4"}
```

Comment:
Are these necessary/desirable?

Reply:
No, these should not be necessary.

Reply:
Removed and confirmed locally that list numbering is correct. Forced numbering is not necessary.